<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mouse event filters</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QSlider" name="senseSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>127</number>
        </property>
        <property name="value">
         <number>64</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Key event filters</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="mtrans">
          <property name="text">
           <string>Translations</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">filterGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mrot">
          <property name="text">
           <string>Rotations</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">filterGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mdom">
          <property name="text">
           <string>Dominant axis</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">filterGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="msens">
          <property name="text">
           <string>Sensitivity</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">filterGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="3" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="ktrans">
          <property name="text">
           <string>Translations</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">filterGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="krot">
          <property name="text">
           <string>Rotations</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">filterGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kdom">
          <property name="text">
           <string>Dominant axis</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">filterGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ksens">
          <property name="text">
           <string>Sensitivity</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">filterGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Sensitivity</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mtrans</tabstop>
  <tabstop>mrot</tabstop>
  <tabstop>mdom</tabstop>
  <tabstop>msens</tabstop>
  <tabstop>ktrans</tabstop>
  <tabstop>krot</tabstop>
  <tabstop>kdom</tabstop>
  <tabstop>ksens</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="filterGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
